src/config/pci-passthrough.md

```diff
@@ -0,0 +1,405 @@
+        02:00.0 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] 300 Series Chipset PCIe Port [1022:43b4] (rev 02) 
+        02:02.0 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] 300 Series Chipset PCIe Port [1022:43b4] (rev 02) 
+        02:03.0 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] 300 Series Chipset PCIe Port [1022:43b4] (rev 02) 
+        02:04.0 PCI bridge [0604]: Advanced Micro Devices, Inc. [AMD] 300 Series Chipset PCIe Port [1022:43b4] (rev 02) 
+        03:00.0 USB controller [0c03]: ASMedia Technology Inc. ASM1143 USB 3.1 Host Controller [1b21:1343] 
+        04:00.0 Ethernet controller [0200]: Intel Corporation I211 Gigabit Network Connection [8086:1539] (rev 03) 
+        05:00.0 Ethernet controller [0200]: Qualcomm Atheros Killer E2500 Gigabit Ethernet Controller [1969:e0b1] (rev 10) 
+```
+
+This is why the PCI slot you plug in your graphics card matters. Some PCI slots
+may have other peripherals in the same IOMMU group that makes the slot
+inconvienient to pass through. Once you have tweaked this and are fine with any
+extra peripherals that may be unusable on your host, continue to the next step.
+
+I will be passing through group 14.
+
+### Isolating GPU
+
+Find the GPU *and its audio device* you want to pass through to your guest in
+IOMMU group list and get the content inside the second square brackets.
+
+For example the GPU I want to passthrough and its audio device had the following
+lines:
+
+```
+IOMMU Group 14: 
+        07:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] [1002:67df] (rev e7) 
+        07:00.1 Audio device [0403]: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere HDMI Audio [Radeon RX 470/480 / 570/580/590] [1002:aaf0] 
+```
+
+The device ids of the devices I would want to passthrough would be: `1002:67df`
+and `1002:aaf0`.
+
+> Note: If you have a host with two identical GPUs, you will have to follow
+> different instructions to passthrough your GPU See: [Using identical guest and
+> host GPUs
+> (Archwiki)](https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF#Using_identical_guest_and_host_GPUs)
+
+Add the kernel parameter: `vfio-pci.ids=1002:67df,1002:aaf0` to your bootloader
+config and regenerate the config for your bootloader like we did previously. If
+you had other components in your IOMMU group, do not add them to
+`vfio-pci.ids=`. This kernel parameter should only have the graphics card and
+its audio device even if you end up passing through a USB controller or
+something else later on.
+
+### Checking
+
+After rebooting, check if the `vfio-pci` driver has been properly added to your
+GPU and it's audio device by running the following command:
+
+```
+$ lspci -nnk -d 1002:67df 
+07:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] [1002:67df] (rev e7) 
```

Comment:
Again, too much command output; i feel the sentence beginning on L327 should be sufficient.